subsys/testsuite/ztest/Kconfig

```diff
@@ -35,23 +34,35 @@ config ZTEST_ASSERT_VERBOSE
 	  1 Write file, line number, function and reason for failed assertions
 	  2 Log also successful assertions
 
+config ZTEST_THREAD_PRIORITY
+	int "Testing thread priority"
+	default -1
+	help
+	  Set priority of the testing thread. Default is -1 (cooperative).
+
+config ZTEST_RETEST_IF_PASSED
+	bool "Reset the board to test again if the test passed"
+	select REBOOT
```

Comment:
Does REBOOT has more dependencies that need to be covered here?

Reply:
If so, these should be included under REBOOT, not here.

Reply:
As far as I know there are no more dependencies but if there are they do indeed belong under REBOOT.